src/write/unit.rs

```diff
@@ -1317,18 +1338,35 @@ mod convert {
                         .get_offset(context.loclists_base, index)?;
                     AttributeValue::LocationListsRef(offset)
                 }
-                read::AttributeValue::RangeListsRef(val) => AttributeValue::RangeListsRef(val),
+                read::AttributeValue::RangeListsRef(val) => {
+                    let base_address = 0; // FIXME get from DW_AT_low_pc attribute in the DW_TAG_compile_unit entry
+                    let iter = context.dwarf.ranges.ranges(
+                        val,
+                        from_unit.version(),
+                        from_unit.address_size(),
+                        base_address,
+                        &context.dwarf.debug_addr,
+                        context.addr_base,
+                    )?;
+                    if let Some(range_list) = RangeList::from(iter, context.convert_address)? {
+                        let range_id = context.ranges.add(range_list);
+                        AttributeValue::RangeListsRef(range_id)
+                    } else {
+                        return Err(ConvertError::InvalidAddress);
+                    }
+                }
                 read::AttributeValue::DebugRngListsBase(_base) => {
                     // We convert all range list indices to offsets,
                     // so this is unneeded.
                     return Ok(None);
                 }
                 read::AttributeValue::DebugRngListsIndex(index) => {
-                    let offset = context
+                    let _offset = context
                         .dwarf
                         .ranges
                         .get_offset(context.rnglists_base, index)?;
-                    AttributeValue::RangeListsRef(offset)
+                    //AttributeValue::RangeListsRef(offset)
+                    unimplemented!();
```

Comment:
Please implement. This should be as simple as passing the offset to the same code that handles `read::AttributeValue::RangeListsRef`.